lmdeploy/vl/model/yi.py

```diff
@@ -83,10 +84,8 @@ def init_yi_model():
 class YiVisionModel(LlavaVisionModel):
     """Yi visual model."""
 
-    def __init__(self, model_path, device='cuda:0'):
-        self.model_path = model_path
-        self.device = device
-        self.build_model()
+    def __init__(self, model_path, vision_config: VisonConfig = None):
+        super().__init__(model_path=model_path, vision_config=vision_config)
```

Comment:
`LlavaVisionModel` does not have keyword argument `vision_config` in its init method`
as 
```
Traceback (most recent call last):
  File "wscript/test_vl_pipeline.py", line 11, in <module>
    pipe = pipeline(model_path, vision_config=vision_config, backend_config=TurbomindEngineConfig(tp=2))
  File "/projects/lmdeploy/lmdeploy/api.py", line 94, in pipeline
    return pipeline_class(model_path,
  File "/projects/lmdeploy/lmdeploy/serve/vl_async_engine.py", line 17, in __init__
    self.vl_encoder = ImageEncoder(model_path, vision_config)
  File "/projects/lmdeploy/lmdeploy/vl/engine.py", line 69, in __init__
    self.model = load_vl_model(model_path)
  File "/projects/lmdeploy/lmdeploy/vl/model/builder.py", line 32, in load_vl_model
    return YiVisionModel(model_path)
  File "/projects/lmdeploy/lmdeploy/vl/model/yi.py", line 88, in __init__
    super().__init__(model_path=model_path, vision_config=vision_config)
TypeError: __init__() got an unexpected keyword argument 'vision_config'
```

Reply:
fixed